Remotion/Web/ClientScript/Scripts/TreeView.ts

```diff
@@ -53,11 +60,22 @@ class WebTreeView
             if (!ev.isTrusted)
               return;
 
+            if (ev.target === linkElement)
+              return;
+
             linkElement.dispatchEvent (new PointerEvent (ev.type, ev));
             ev.preventDefault();
             ev.stopPropagation();
           });
         }
+
+        const expanderElement = node.querySelector(':scope > a:first-child');
```

Comment:
Consistency question. In line 37, we have a cast to `Nullable<HTMLElement>`. Why do we not have this here, too?

Reply:
Removed the cast above